packages/commonwealth/client/scripts/views/components/sidebar/sidebar_section.tsx

```diff
@@ -49,7 +49,7 @@ const SubSection = (props: SubSectionAttrs) => {
   );
 };
 
-const SubSectionGroup = (props: SectionGroupAttrs) => {
+export const SubSectionGroup = (props: SectionGroupAttrs) => {
```

Comment:
Github CI => `Declare only one React component per file`

TBH I think we should disable this lint rule

Reply:
Although this warning is annoying, because it makes you fix the file you only changed one line, in overall I think this rule is very helpful because it doesnt allow putting too much code in one file. In this particular case I will probably skip this file as there are 3 or 4 components and I am tight in time